ruby/ruby_releases/ruby_benchmarks/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
As you said we must call this script `build`, we have another build script in all these repositories so any another particular name to use?